spec/features/bulkrax_export_spec.rb

```diff
@@ -2,34 +2,37 @@
 
 require "rails_helper"
 
-RSpec.describe "Bulkrax export", clean: true do
+RSpec.describe "Bulkrax export", clean: true, type: :feature, slow: true do
   let(:user) { create(:user, email: "test@example.com") }
   # let! is needed below to ensure that this user is created for file attachment because this is the depositor in the CSV fixtures
-  let!(:depositor) { create(:user, email: "batchuser@example.com") }
+  let!(:depositor) { build_stubbed(:user, email: "batchuser@example.com") }
+
+  let(:export_source) { work.class.to_s }
+  let(:limit) { 0 }
+
   let(:exporter) do
     create(:bulkrax_exporter_worktype,
            user: user,
            field_mapping: Bulkrax.field_mappings["HykuAddons::CsvParser"],
            parser_klass: "HykuAddons::CsvParser",
-           export_source: work.class.to_s,
-           limit: 0)
+           export_source: export_source,
+           limit: limit)
   end
-  let(:work) { create(:fully_described_work, user: depositor.email) }
-  let(:account) { create(:account) }
 
-  before do
-    # Make sure default admin set exists
-    AdminSet.find_or_create_default_admin_set_id
-  end
+  let(:account) { build_stubbed(:account) }
 
   after do
```

Comment:
Whilst we're doing this big edit, we might as well clean up stuff thats obviously never going to be done. :)